### PR TITLE
[WIP] Remove Ember.ContainerView.

### PR DESCRIPTION
**Pull request by rwjblue**
I started working on this a few days ago, but haven't had a chance to circle around to it.

### CODE REVIEW

**Comment by rwjblue** [comment]
@HeroicEric - Want another challenge?

**Comment by HeroicEric** [comment]
heh, I actually tried this one already the other day but got stuck. I can try though :smile:

File: packages/ember-views/tests/views/view/remove_test.js [hunk 1]
```diff
@@ -10,44 +9,8 @@ import viewKeyword from 'ember-htmlbars/keywords/view';
 import { objectAt } from 'ember-runtime/mixins/array';
 
 
-// .......................................................
-// removeChild()
```
Comment (HeroicEric) [inline packages/ember-views/tests/views/view/remove_test.js]: @rwjblue was this meant to be removed? If so, should `View#removeAllChildren` and friends have the same fate?

Comment (HeroicEric) [inline packages/ember-views/tests/views/view/remove_test.js]: @rwjblue was this meant to be removed? If so, should `View#removeAllChildren` and friends have the same fate?

Comment (rwjblue) [inline packages/ember-views/tests/views/view/remove_test.js]: I was uncertain if removeAllChildren is a "normal" view method or just for container view.

Comment (HeroicEric) [inline packages/ember-views/tests/views/view/remove_test.js]: @rwjblue `removeAllChildren` is from the `LegacyViewSupport` mixin but it uses `removeChild` https://github.com/rwjblue/ember.js/blob/remove-container-view/packages/ember-views/lib/mixins/legacy_view_support.js#L35-L39

Comment (rwjblue) [inline packages/ember-views/tests/views/view/remove_test.js]: kk, seems good lets kill it for now and we can see where that puts us

**Pull request closed without merging**